src/components/Notification/types.ts

```diff
@@ -0,0 +1,29 @@
+import { iconTypes } from "../../components/Icon/collection";
+
+export interface NotificationProps {
+        /**
+         * The notification ID will generated if not assigned
+         */
+        id?: string;
+    
+        /**
+         * The title to display in notification
+         */
+        title?: string
+    
+        /**
+         * The message to display in notification
+         */
+        message?: string;
+
+        /**
+         * set an icon to show inside the notification
+         * import { iconTypes } from "../../components/Icon/collection"
+         */
+        icon?: iconTypes;
+
+        /**
+         * set if notification is active/viewable
+         */
+        active?: boolean;
```

Comment:
For this Boolean would you consider that `isVisible` or `isShowing` could be more intuitive? 

My old boss would insist that all Boolean should be isSomething or hasSomething but i get that things like disabled and visible seem more intuitive in some cases

Reply:
isVisible sounds better